src/PKSim.Core/Snapshots/Mappers/ExpressionContainerMapper.cs

```diff
@@ -92,7 +77,7 @@ private bool shouldMapContainer(IParameter expressionParameter, TransporterExpre
 
          var species = individual.Species.Name;
          var transporterExpressionContainer = expressionContainer.DowncastTo<TransporterExpressionContainer>();
-         var membraneLocation = ModelValueFor(snapshot.MembraneLocation, _defaultMembraneLocation);
+         var membraneLocation = ModelValueFor(snapshot.MembraneLocation, MembraneLocation.Basolateral);
```

Comment:
can u explain this change?

Reply:
There is no member _defaultMembraneLocation. This member was avoid exporting the membrane location and reduced the number of entry. But the logic was not ok. Right now MembraneLocation will be exported as soon as the RelExp is !=0. I cannot enforce the presence of the member however because the same object is used for non transporter container
The default value for a contaienr in DB is in general MembraneLocation.Basolateral that;s why I am using it/
Alternatively, we could throw a warning if the location is not set. Find either way

Reply:
ok. I thik we don't need a warning in this case.